### ISSUE TITLE
I want to use custom camera

### ISSUE THREAD

**Issue by ibenettaleb** [WebsiteBeaver/react-native-document-scanner-plugin#3]
Hello ;-)

Thank you very much for your fantastic plugin.

I'm simply wondering whether I may utilize the **custom camera** that I made.

- Take a look at the following screenshot:

<img src="https://user-images.githubusercontent.com/28574987/187429966-2052acac-1388-4b35-b5a7-882ac5ab946b.jpg"  width="45%">

- Give me this screen after that:

<img src="https://user-images.githubusercontent.com/28574987/187430070-7880af66-f28f-4358-9fce-97e7f0c612a3.jpg"  width="45%">

 Thanks for considering my request.

**Comment by dmarcs** [comment]
That would probably not be too difficult for Android. The problem is that iOS uses the native document scanner, so I can't customize that. It would only be possible if I use Vision instead.

**Comment by ibenettaleb** [comment]
I appreciate your reply. How can I include it in Android?
Or will you include that in the future?

**Comment by dmarcs** [comment]
I'll consider it if time allows. Right now you can't do this with Android.

**Comment by ibenettaleb** [comment]
OK, Thank You

**Comment by comertcimen** [comment]
Is the answer same for using the cropping/image editing screen for without camera usage?
e.g. using share functionality to import an image from third party apps and passing image path to the cropper.

**Comment by dmarcs** [comment]
Yeah it's basically the same. In order to make that work I would need to skip the camera capture on Android, and send the cropper an image. On iOS it's possible, but tricky because right now this plugin uses VisionKit.

**Comment by deepakv-z14** [comment]
Yes this will be a great addition to this lib

**Comment by harbolaez** [comment]
+1 this would be a great addition or if you can expose the cropping functionality on Adnroid would be great!

**Comment by paramrana78** [comment]
+1 
Team, I'm also looking for something similar, just to show the rectangular mask on camera without any functionality.
